changelog.md

```diff
@@ -0,0 +1,33 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
+
+Find your version by looking for the file for your mod-loader:
+
+-   Entropy-`<mod-version>`+mc`<mc-version>`.jar
+
+## [1.13] - 2025-03-01
+
+Special to the Entropy Contributors: bl4ckscor3, Kanawanagasaki and slicedlime! <3
+
+Available versions:
+-   Minecraft 1.20.1
+
+### New Events:
+-   Nothing, Silence, Rainbow trails, Spawn Rainbow sheep, jeb_, Armor trim Event
+
+### Changed/Fixed
+
+-   Increased base event and timer duration settings to accept up to 180 seconds.
+-   One Heart event has been revamped and is now called "Half-Hearted".
+-   Improved Stuttering, Midas Touch and So Sweet! Events.
+-   Entropy Overlay now hides when Debug F3 is being shown.
+-   Improved german translations.
+-   Added brazilian portuguese translations.
+-   Small fixes to various events.
+
+## [1.12] - 2023-03-14
+
+For releases prior to Entropy 1.12, please check the changelog attached to the version download page.
```

Comment:
The 1.12 changelog is not available in this file, so it is actually "prior to Entropy 1.13" where people have to check somewhere else.